packages/data-types/test/types/v1/any-spec.ts

```diff
@@ -0,0 +1,43 @@
+import AnyType from '../../../src/types/v1/any';
+import { FieldTypeConfig } from '../../../src/interfaces';
+
+describe('Boolean V1', () => {
```

Comment:
This file references Boolean in a few places

```suggestion
describe('Any V1', () => {
```